smart-contracts/dev-environments/remix/get-started.md

```diff
@@ -1,167 +1,89 @@
 ---
-title: Use the Polkadot Remix IDE
-description: Explore the smart contract development and deployment process on Asset Hub using Remix IDE, a visual IDE for blockchain developers.
+title: Use the Remix IDE on Polkadot Hub
+description: Explore the smart contract development and deployment process on Polkadot Hub using Remix IDE, a visual IDE for blockchain developers.
 categories: Smart Contracts, Tooling
 ---
 
 # Remix IDE
 
---8<-- 'text/smart-contracts/polkaVM-warning.md'
-
-<div class="grid cards" markdown>
--   :octicons-code-16:{ .lg .middle } __Deploy NFTs Using Remix IDE__
-
-    ---
-
-    Mint your NFT on Polkadot's Asset Hub. Use PolkaVM and OpenZeppelin to bring your digital asset to life with Polkadot Remix IDE.
-
-    <br>
-    [:octicons-arrow-right-24: Get Started](/smart-contracts/cookbook/smart-contracts/deploy-nft/){target=\_blank}
-
--   :octicons-code-16:{ .lg .middle } __Deploy ERC20s Using Remix IDE__
-
-    ---
-
-    Mint your custom ERC-20 token on Polkadot's Asset Hub. Leverage PolkaVM and Polkadot Remix IDE to bring your blockchain project to life.
-
-    <br>
-    [:octicons-arrow-right-24: Get Started](/smart-contracts/cookbook/smart-contracts/deploy-erc20/){target=\_blank}
-</div>
-
-!!! warning
-    The Polkadot Remix IDE's contract compilation functionality is currently limited to Google Chrome. Alternative browsers are not recommended for this task.
-
 ## Overview
 
-Remix IDE is a robust browser-based development environment for smart contracts. This guide will walk you through the essentials of the [Polkadot Remix IDE](https://remix.polkadot.io/){target=\_blank} to understand the processes of compiling, developing, and deploying smart contracts on Asset Hub.
+Remix IDE is a robust browser-based development environment for smart contracts. This guide will walk you through the essentials of the [Remix IDE](https://remix.ethereum.org/){target=\_blank} to understand the processes of compiling, developing, and deploying smart contracts on Polkadot Hub.
 
 ## Prerequisites
 
 Before getting started, ensure you have:
 
-- A web browser with [Talisman](https://talisman.xyz/){target=\_blank} extension installed.
+- A web browser with [MetaMask](https://metamask.io/){target=\_blank} extension installed.
```

Comment:
```suggestion
- A web browser with the [MetaMask](https://metamask.io/){target=\_blank} extension installed.
```

It sounds weird but the way this is phrased it should either have "the" before MetaMask since we're specifically calling out the extension or it should just be "A web browser with MetaMask installed."